seed/models/columns.py

```diff
@@ -53,8 +53,10 @@ class Column(models.Model):
     # Do not return these columns to the front end -- when using the tax_lot_properties
     # get_related method.
     EXCLUDED_COLUMN_RETURN_FIELDS = [
+        'created',
```

Comment:
The inclusion of `created` and `updated` within 
- EXCLUDED_COLUMN_RETURN_FIELDS
- COLUMN_EXCLUDE_FIELDS
- EXCLUDED_MAPPING_FIELDS

...all seemed appropriate. Can anybody else confirm this?

Reply:
yeas this sounds right  based on our discussions. The property.updated and propety.created are not included in the hash, nor should any created/updated.